src/main/java/org/opensearch/security/securityconf/impl/v6/ConfigV6.java

```diff
@@ -248,6 +248,15 @@ public String toString() {
                 + "]";
         }
 
+        @JsonAnySetter
+        public void unknownPropertiesHandler(String name, Object value) {
```

Comment:
If the property name doesn't match an error is not thrown?  Lets add/modify tests to 1) make sure the previous behavior is unchanged for unknown properties, and 2) the new behavior logs that message.

Example test case for capturing the deprecation log message:
https://github.com/opensearch-project/security/blob/b905999f81093fc96db8371df5f96badabb9ff38/src/test/java/org/opensearch/security/setting/DeprecatedSettingsTest.java#L30-L31